src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -652,66 +652,6 @@ function copyOptions() {
     feedbackRecipientUpdateVisibilityOptions($currRecipient);
 }
 
-function enableRow(elem, row) {
```

Comment:
Aiyo, I didn't remove these when moving them to `visibilityOptions.js` in #5642 =(